src/type/definition.js

```diff
@@ -506,6 +506,11 @@ function defineFieldMap<TSource, TContext>(
       `${type.name}.${fieldName} field type must be Output Type but ` +
       `got: ${String(field.type)}.`
     );
+    invariant(
+      isValidResolver(field.resolve),
```

Comment:
"!field.resolve" means we will accept any "falsey" value such as 0 or the empty string. Is that what we want?